test-support/page-object/properties/click-on-text.js

```diff
@@ -45,11 +51,22 @@ export function clickOnText(selector, options = {}) {
     isDescriptor: true,
 
     value(textToClick) {
-      wait().then(() => {
-        var actualSelector = findChildElement(this, selector, textToClick, options) || findElement(this, selector, textToClick, options);
+      var context = getContext(this);
+
+      if (context) {
+        Ember.run(() => {
+          var fullSelector = actualSelector(this, selector, textToClick, options);
+
+          context.$(fullSelector).click();
+        });
+      } else {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["`wait`" is defined but never used](https://www.codacy.com/app/san650/ember-cli-page-object/file/2020892486/issues/source?bid=2903707&fileBranchId=3076256#l63)